### ISSUE TITLE
Simple Row Reordering example is not working

### ISSUE THREAD

**Issue by FireMuse** [6pac/SlickGrid#1037]
### Describe the bug

[https://6pac.github.io/SlickGrid/examples/example9-row-reordering-simple.html] 

The example for row reordering is not working.

### Reproduction

Load [https://6pac.github.io/SlickGrid/examples/example9-row-reordering-simple.html](url). Try to follow the instructions. It doesn't work.

### Which Framework are you using?

Angular

### Environment Info

```shell
| Executable          | Version |
| ------------------- | ------- |
| (framework used)    | VERSION |
| SlickGrid | VERSION |
| TypeScript          | VERSION |
| Browser(s)          | VERSION |
| System OS           | VERSION |
```

### Validations

- [X] Follow our [Code of Conduct](https://github.com/6pac/SlickGrid/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [Wikis](https://github.com/6pac/SlickGrid/wiki).
- [X] Check that there isn't [already an issue](https://github.com/6pac/SlickGrid/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/6pac/SlickGrid/discussions).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by ghiscoding** [comment]
The other [example9-row-reordering](https://6pac.github.io/SlickGrid/examples/example9-row-reordering.html) is working, you can take a look at that one and perhaps you can contribute a fix to the simple one

**Comment by ghiscoding** [comment]
fixed by PR #1042, the problem was that the Example wasn't properly requesting the drag column from the plugin, it was just a random column with drag icons and we can't simply do that. We really need to let the plugin create the drag column for us, we then get it from `columns.unshift(rowMovePluging.getColumnDefinition())` and that will then work correctly

**Comment by ghiscoding** [comment]
Edit, I've now also fixed the drag row(s) to the recycle bin as well in the same example, and so dragging row(s) will now delete them from the grid.
